lib/portage/tests/resolver/test_virtual_slot.py

```diff
@@ -71,7 +71,13 @@ def testVirtualSlotUpdate(self):
             },
             "virtual/jdk-1.7.0": {
                 "SLOT": "1.7",
```

Comment:
Bah,
So I figured out why I was having problems trying to write a test case in our 2.3.75 branch:
https://chromium-review.googlesource.com/c/chromiumos/third_party/portage_tool/+/2936978

We  mark EAPI 0 as deprecated.

Since none of these test packages have their EAPI specified, they default to 0. That means that these packages have the `EAPI.deprecated` mask which in turn makes the `visible` property false.

Changing `ResolverPlayground._create_ebuilds` to default to EAPI 5 in our fork makes the test pass.